java/arcs/core/entity/EntityBase.kt

```diff
@@ -147,6 +150,9 @@ open class EntityBase(
     /** Returns the [FieldType] for the given collection field, or null if it does not exist. */
     private fun getCollectionTypeOrNull(field: String) = schema.fields.collections[field]
 
+    /** Returns true if the collection has the [FieldType]. */
+    protected fun hasCollectionField(field: String) = getCollectionTypeOrNull(field) != null
```

Comment:
could maybe be inline?

Reply:
do you mean an `inline` function?

Reply:
yes, what do you think? it's simply an expression on an existing method, so seems like a good use.

Reply:
I gave it a try, but it won't work: 
`public-API inline function cannot access non-public-API 'private final fun getSingletonTypeOrNull(field: String): FieldType? defined in arcs.core.entity.EntityBase'`